challenge-ais-design/readme.md

```diff
@@ -0,0 +1,54 @@
+Challenge - AIS design
+=====
+
+> The automatic identification system, or AIS, transmits a ship’s position so that other ships are aware of its position. The International Maritime Organization and other management bodies require large ships, including many commercial fishing vessels, to broadcast their position with AIS in order to avoid collisions. Each year, more than 400,000 AIS devices broadcast vessel location, identity, course and speed information. Ground stations and satellites pick up this information, making vessels trackable even in the most remote areas of the ocean.[1]
+
+
+Task
+-----
+
+Design an AIS representation for Orca that balances the given considerations. The goal of the representation is to show other vessels in the user's marine chart to help the user:
+
+a) Avoid collisions, close encounters, and respect right of way[2]
+b) Understand the general traffic conditions at a given area prior to arrival/entry in the area
+
+Details
+-----
+
+### The user persona
+
+The user is a recreational boater. The user understands the principles of AIS. The user conducts a combination of smaller daily boat trips around his/her harbour, and longer distance trips – such as crossing the British Channel.
+
+### Considerations
+
+1. Vessel types should be differentiated to allow the user to easily identify and proactively adjust to respect right of way
```

Comment:
should give some hints on what "Vessel types" are? or enough to reference MT and let them figure it out?